R/build_markdown.R

```diff
@@ -96,7 +96,13 @@ build_markdown <- function(path = ".", rebuild = FALSE, quiet = FALSE, slug = NU
         error   = error
       )
     }
-    handout <- getOption("sandpaper.handout", default = FALSE)
+
+    handout <- this_metadata$get()[["handout"]]
+
+    # produces the default headings, challenges, code, etc
+    # TODO: this needs improving to allow users to choose what to include based
+    # on a yaml list
+    handout <- !is.null(handout) || handout
```

Comment:
The checks are building because of these two lines and I realise why. My apologies because I was navigating when we did this. 

this pattern of `!is.null(thing) || thing` is only useful if `thing` only takes boolean. In this case we need a the inline ifelse:

```suggestion
    handout <- if (is.null(handout)) FALSE else handout
```

Reply:
I'm going to commit it to get this passing